### ISSUE TITLE
[L5] Notification is sent to SDC if Upload/Download API performance drops below 70% of average rate for an hour

### ISSUE THREAD

**Issue by sdhoyt** [IMAP-Science-Operations-Center/sds-data-manager#89]
**Summary**
A notification system should be setup so that the SDC is alerted if data transfer rates drop below a defined threshold

**Parent Requirement**
SOC-SDC-L4-70 | SDC Data Transfer Monitoring | The SDC shall be capable of monitoring data transfer rates. | Requirement | Necessary to ensure IMAP data is available to the entire science community without undue delay. | SOC-L3-57 | Test | SIT-2SIT-5 | SIT-5
-- | -- | -- | -- | -- | -- | -- | -- | --

**Tasks**
- [x] #160

**Comment by greglucas** [comment]
@sdhoyt are we able to close this issue? The linked issues have been addressed and we have the monitoring now.

**Comment by sdhoyt** [comment]
Yeah, thanks for reminding me about this. This was closed in SIT-3
